src/platform/utilities/api/index.js

```diff
@@ -49,6 +50,7 @@ function isJson(response) {
 export function apiRequest(resource, optionalSettings = {}, success, error) {
   const baseUrl = `${environment.API_URL}/v0`;
   const url = resource[0] === '/' ? [baseUrl, resource].join('') : resource;
+  const csrfToken = Cookies.get('X-CSRF-Token');
```

Comment:
Here are some files in the app code that are making direct `fetch()` or `XMLHTTPRequest` calls without using the `apiRequest` helper.  Do we know what the impact of the CSRF change on these apps will be? Should we refactor them to ensure that the requests are made with a CSRF token?

![image](https://user-images.githubusercontent.com/2008881/77205503-3be7ee80-6ab2-11ea-83c6-7e1b4c8043b8.png)

![image](https://user-images.githubusercontent.com/2008881/77205693-9d0fc200-6ab2-11ea-97c6-7cc5aeb4250d.png)

Reply:
This also leads into the question of do we want to discourage people from making direct `fetch()` and `XMLHTTPRequest` calls in the future? If so, maybe we can use a linting rule?

Reply:
https://github.com/department-of-veterans-affairs/va.gov-team/issues/224

We can revisit the fetch refactoring later. I would like to get rid of the apiRequestHelper and provide chainable boilerplate that everyone uses with fetch.